src/languages/en.ts

```diff
@@ -741,8 +741,8 @@ export default {
         reason: 'Reason',
         holdReasonRequired: 'A reason is required when holding.',
         expenseOnHold: 'This expense was put on hold. Review the comments for next steps.',
-        confirmApprove: 'Confirm what to approve',
-        confirmApprovalAmount: 'Approve the entire report total or only the amount not on hold.',
+        confirmApprove: 'Confirm approval amount',
+        confirmApprovalAmount: 'Approve what's not held up, or approve the entire report.',
```

Comment:
@jamesdeanexpensify I think once you fix this string the errors will go away (like I pointed out in the other one, you need to use double quotes if the sentence contains a single quote)